contracts/LoanManager.sol

```diff
@@ -246,8 +249,11 @@ contract LoanManager is Restricted {
             (loanAmount, interestAmount) = calculateLoanValues(product, loan.repaymentAmount);
             uint disbursementTime = loan.maturity - product.term;
 
+            LoanState loanState =
```

Comment:
suggest moving this duplicate logic into a common function.

Reply:
I was wondering to move the whole  result generation into one function (for one loan). But I wasn't sure passing uint[10] arrays on heap is a good idea. Although this is an external view only fx so might be OK.
What you think?

Reply:
OK, just pushed it with a new common function: `getLoanTuple`